### DISCUSSION TITLE
How to draw a dotted line

### DISCUSSION THREAD

**Discussion by huhui123123** [fabricjs/fabric.js#6949] [Q&A]
How to draw the dotted line, thank you

**Comment by asturur** [comment]
Please have a look at the docs before posting basic questions.
http://fabricjs.com/docs/fabric.Object.html#strokeDashArray
Experiment with this setting.

**Reply by huhui123123** [reply]
Hello, because my English is not good, many do not understand, can get your help, thank you very much

**Reply by asturur** [reply]
try to set 'strokeDashArray = [1, 1]' on your line
